### PR TITLE
Create pairings

### LINKED ISSUE #23: Create 'createPairing' GraphQL mutation endpoint

### LINKED ISSUE #29: Create 'createPairings' GraphQL mutation endpoint

### PR DESCRIPTION
**Pull request by msimon42**
### Sidebar Checklist

- [x ] Request reviewers
- [x ] Assign yourself and other contributors
- [x ] Link to project

### Issues Resolved
Resolves #23 
Resolves #29 

### Problem Addressed
There was literally no way to create a pairing using our API, and now there is!!
### Solution Implemented
Created graphql mutation to create new pairing
### Other Notes
🥇

### CODE REVIEW

File: app/graphql/mutations/pairings/create_pairings.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+module Mutations
+  module Pairings
+    class CreatePairings < ::Mutations::BaseMutation
+      argument :pairer_id, Integer, required: true
+      argument :pairee_id, Integer, required: true
+      argument :date, String, required: true
+      argument :time, String, required: true
+      argument :notes, String, required: false
```
Comment (DanielEFrampton) [inline app/graphql/mutations/pairings/create_pairings.rb]: Matt, nice work putting this together. I noticed that there's a difference between how you wrote the mutation's arguments and what the FE has in its query, which might be related to something we figured out yesterday. Our initial assumption was that pairings were created and saved to the database when someone clicks "Book Session," but we discovered yesterday that the actual case is that the pairings are created with `pairee` as `null` when a user defines their availability, and then they are _updated_ when another user books a session with the pairer. So the mutation in `src/queries/index.js` is like this:
```ruby
mutation {
    createPairings(
      pairings: ${pairingsString}
    ) {
      pairer
      date
      time
    }
  }
```
Which doesn't have a `pairee_id` or `notes` as arguments. Can you revise it accordingly?

Comment (DanielEFrampton) [inline app/graphql/mutations/pairings/create_pairings.rb]: Matt, nice work putting this together. I noticed that there's a difference between how you wrote the mutation's arguments and what the FE has in its query, which might be related to something we figured out yesterday. Our initial assumption was that pairings were created and saved to the database when someone clicks "Book Session," but we discovered yesterday that the actual case is that the pairings are created with `pairee` as `null` when a user defines their availability, and then they are _updated_ when another user books a session with the pairer. So the mutation in `src/queries/index.js` is like this:
```ruby
mutation {
    createPairings(
      pairings: ${pairingsString}
    ) {
      pairer
      date
      time
    }
  }
```
Which doesn't have a `pairee_id` or `notes` as arguments. Can you revise it accordingly?

**Comment by DanielEFrampton** [comment]
Side note: it looks like I mistakenly made issue #23 when there's actually no such mutation; instead of resolving it, can we delete it?

File: spec/factories/users.rb [hunk 2]
```diff
@@ -8,5 +8,6 @@
     email { Faker::Internet.email }
     image { Faker::LoremFlickr.image }
     phone_number { "4233162121" }
+    firebase_id {'lmfao1233444'}
```
Comment (rer7891) [inline spec/factories/users.rb]: Please update line 11 with: firebase_id { Faker::Alphanumeric.alphanumeric(number: 28 )}

**Pull request closed without merging**